Documentation/project-docs/target-dotnetcore-with-msbuild.md

```diff
@@ -0,0 +1,119 @@
+
+- Select ".NET Framework 4.6" and "ASP.NET Core 1.0"
+
+  ![Portable targets dialog](pcl-targets-dialog-net46-aspnetcore10.png)
+
+- In the "Library" tab of the project properties, click on the "Target .NET Platform Standard" link, and click "Yes" in the dialog that is shown
+- In the `project.json` file:
+    - Change the version number of the `NETStandard.Library` package to `1.5.0-rc2-24027` (this is the .NET Core RC2 version of the package).
+    - Add the below `imports` definition inside the `netstandard1.5` framework definition.  This will allow your project to reference .NET Core compatible
+      NuGet packages that haven't been updated to target .NET Standard
+
+        ```json
+        "netstandard1.5": {
+            "imports": [ "dnxcore50", "portable-net452" ]
+        }
+        ```
+
+- You may see an error in the Visual Studio error list saying "Platform target 'AnyCPU' is not supported by one or more of the project's targets."  If so,
+you can suppress the error by editing the `.csproj` file and removing the line with the `ProjectTypeGuids` property  
+
+Creating a .NET Core console application
+========================================
+Building a console application for .NET Core requires some customization of the MSBuild build process.  A good option to start with is
+[coretemplate](https://github.com/mellinoe/coretemplate).  This is a basic project which brings in MSBuild targets for .NET Core.  
+
+It is also possible to start by creating a project in Visual Studio and modify it to target .NET Core.  The instructions below show the minimal steps to get this working.
+In contrast to coretemplate, a project created this way:
+
+- Won't include configurations for targeting Linux and Mac OS
+- Will use the CoreRun instead of the CoreConsole host.  This means that you will need to run the app via `CoreRun MyApp.exe` instead of just `MyApp.exe`.
+To use the CoreConsole host, you need to reference the `Microsoft.NETCore.ConsoleHost` package instead of `Microsoft.NETCore.TestHost`, and after building rename `MyApp.exe`
+to `MyApp.dll` and rename `CoreConsole.exe` to `MyApp.exe`.  Coretemplate does this in a [common targets file](https://github.com/mellinoe/corebuild/blob/master/coreconsole.targets).
+- Visual Studio may ask you to enable developer mode for Windows 10 when you open the project
+
+Creating a .NET Core console application from Visual Studio
+===========================================================
+
+- Start by following the steps in the previous section to create a library targeting .NET Core
+- Open the project's XML for editing (in Visual Studio, right click on the project -> Unload Project, right click again -> Edit MyProj.csproj)
+    - Remove the `ProjectTypeGuids` property
+    - Add the following property: `<BaseNuGetRuntimeIdentifier>win10</BaseNuGetRuntimeIdentifier>`
+    - Add the following at the end of the file (after the import of `Microsoft.Portable.CSharp.Targets`:
```

Comment:
Leaving a comment saying this enables debugging using the Managed CoreCLR Debugging Engine will help people grok this seemingly voodoo guid and other params.